pkg/proxy/proxier.go

```diff
@@ -273,7 +273,7 @@ func (proxier *Proxier) OnUpdate(services []api.Service) {
 			}
 			info.portalIP = serviceIP
 			info.portalPort = servicePort.Port
-			info.publicIPs = service.Spec.PublicIPs
+			info.loadBalancerStatus = *service.Status.LoadBalancer.DeepCopy()
```

Comment:
I'd rather skip having to handle error and the interface{} upcast...   But if you disagree then I will do it.  If so should I call conversion.DeepCopy directly, or should I have LoadBalancerStatusDeepCopy call conversion.DeepCopy?